docs/userguide/modules/getting-started/partials/deploy-db-postgresql.adoc

```diff
@@ -0,0 +1,75 @@
+== Create a PostgreSQL database instance
+
+The application is going to use a PostgreSQL database instance. 
+
+To create a database instance, you need to create the following resources:
+
++++ <details><summary> +++
+`Deployment` to run the actual database instance
++++ </summary><div> +++
+[source,yaml]
+----
+include::attachment$postgresql-deployment.yaml[tag=db-deployment]
+----
++++ </div></details> +++
+
++++ <details><summary> +++
+`Secret` to store database credentials
++++ </summary><div> +++
+[source,yaml]
+----
+include::attachment$postgresql-deployment.yaml[tag=db-secret]
+----
++++ </div></details> +++
+
++++ <details><summary> +++
+`Service` to provide means to access to the database
++++ </summary><div> +++
+[source,yaml]
+----
+include::attachment$postgresql-deployment.yaml[tag=db-service]
+----
++++ </div></details> +++
+
+For convenience, you can run this command to create the database\'s resources:
+[source,bash,subs="attributes,macros"]
+----
+kubectl apply -f link:{attachmentsdir}/postgresql-deployment.yaml[pass:a[{site-url}/{page-component-name}/{page-module}/{attachmentsdir}/postgresql-deployment.yaml]] -n {quickstart-namespace}
+----
+
+To verify that the database instance is up and running you can check the respective pod in `{quickstart-namespace}` namespace and see it `Running`:
+[source,bash,subs="attributes"]
+----
+kubectl get pods -n {quickstart-namespace}
+----
+----
+NAME                                           READY   STATUS    RESTARTS   AGE
+spring-petclinic-postgresql-6db5594876-4556g   1/1     Running   0          16m
+----
+
+The database has been created and is empty at this stage. Its schema and a sample data set will be initialized by the application.
+
```

Comment:
Suggestion: Add L74 after L51.

This completes the procedure. Additional information such as Notes and Important admonitions can follow after this.